app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const app = express();
+const {getTopics} = require("./controllers/controller.js");
+
+app.use(express.json());
```

Comment:
You won't need this until you have a request body but I understand you wanting it set up now.